hibernate6/pom.xml

```diff
@@ -0,0 +1,111 @@
+    <artifactId>jackson-datatype-hibernate-parent</artifactId>
+    <version>2.15.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>jackson-datatype-hibernate6</artifactId>
+  <name>Jackson-datatype-hibernate6</name>
+  <packaging>bundle</packaging>
+  <description>Add-on module for Jackson (https://github.com/FasterXML/jackson) to support
+Hibernate (https://hibernate.org/) version 6.x with Jakarta data types.
+  </description>
+  <url>https://github.com/FasterXML/jackson-datatype-hibernate</url>
+  <properties>
+    <!-- Generate PackageVersion.java into this directory. -->
+    <packageVersion.dir>com/fasterxml/jackson/datatype/hibernate6</packageVersion.dir>
+    <packageVersion.package>${project.groupId}.hibernate6</packageVersion.package>
+    <!-- Hibernate with Jakarta Persistence 3.0 -->
+    <hibernate.version>6.1.5.Final</hibernate.version>
+    <osgi.export>${project.groupId}.hibernate6</osgi.export>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>jakarta.transaction</groupId>
+      <artifactId>jakarta.transaction-api</artifactId>
+      <version>2.0.1</version>
+    </dependency>
+
+    <!-- what would be the best scope to use here? -->
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-core</artifactId>
+      <version>${hibernate.version}</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <!--  and for some contributed tests Mockito -->
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.h2database</groupId>
+      <artifactId>h2</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>jakarta.xml.bind</groupId>
+      <artifactId>jakarta.xml.bind-api</artifactId>
+      <version>3.0.1</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <!-- Inherited from oss-base. Generate PackageVersion.java.-->
+        <groupId>com.google.code.maven-replacer-plugin</groupId>
+        <artifactId>replacer</artifactId>
+        <executions>
+          <execution>
+            <id>process-packageVersion</id>
+            <phase>generate-sources</phase>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.moditect</groupId>
+        <artifactId>moditect-maven-plugin</artifactId>
+        <inherited>true</inherited>
+        <configuration>
+          <jvmVersion>11</jvmVersion>
+        </configuration>
+      </plugin>
+
+      <!-- 02-Mar-2017, tatu: Need to publish test jar for 5.2 test to work -->
```

Comment:
I don't think this is needed as we don't include 5.2 test: let's remove `maven-jar-plugin` from here.